RELEASENOTES.md

```diff
@@ -2,10 +2,27 @@
 
 - It is now possible to skip the modification of dependency version numbers when running the Increment Version number workflow or the Create Release workflow
 
+### New Repository Settings
+
+- [`shortLivedArtifactsRetentionDays`](https://aka.ms/algosettings#shortLivedArtifactsRetentionDays) determines the number of days to keep short lived build artifacts (f.ex build artifacts from pull request builds, next minor or next major builds). 1 is default. 0 means use GitHub default.
+- [`longLivedArtifactsRetentionDays`](https://aka.ms/algosettings#longLivedArtifactsRetentionDays) determines the number of days to keep long lived build artifacts (f.ex build artifacts from CI/CD builds). 0 is the default and means use GitHub default.
```

Comment:
If we use GitHubs default, why not just let people control this setting through GitHub? Is there a need for an AL-Go setting?
![image](https://github.com/user-attachments/assets/319ddd12-4624-4451-8a33-8f6efb923d53)

Reply:
I guess we could remove the longLivedArtifactsRetentionDays